src/index.html

```diff
@@ -20,8 +21,437 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating-response">Reviews: 5</p>
+        </div>
+
+        <p class="card__price">
+          <span class="card__price-inscription">Price:</span>
+
+          <span class="card__price-count">$2,199</span>
+        </p>
+
+        <div class="card-button">
+          <a
+            href="#"
+            class="card-button__link"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+      </section>
+      <section class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="product-photo"
+          class="card__photo"
+        />
+
+        <div class="card__top">
+          <h3 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__code">Product code: 195434</p>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating-response">Reviews: 5</p>
+        </div>
+
+        <p class="card__price">
+          <span class="card__price-inscription">Price:</span>
+
+          <span class="card__price-count">$2,199</span>
+        </p>
+
+        <div class="card-button">
+          <a
+            href="#"
+            class="card-button__link"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+      </section>
+      <section class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="product-photo"
+          class="card__photo"
+        />
+
+        <div class="card__top">
+          <h3 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__code">Product code: 195434</p>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating-response">Reviews: 5</p>
+        </div>
+
+        <p class="card__price">
+          <span class="card__price-inscription">Price:</span>
+
+          <span class="card__price-count">$2,199</span>
+        </p>
+
+        <div class="card-button">
+          <a
+            href="#"
+            class="card-button__link"
+            data-qa="hover"
```

Comment:
The `data-qa` attribute value 'hover' is used multiple times, which could be problematic for testing purposes. Ensure each `data-qa` value is unique.